### ISSUE TITLE
API to calculate size of fixed-size types

### ISSUE THREAD

**Issue by danvk** [jDataView/jBinary#44]
While creating custom types, it's sometimes helpful to know the size (in bytes) of a base type. In general this isn't possible to compute (because of dynamically-sized elements), but for simple types like `int32` and objects composed of these simple types, it is.

It would be helpful if there were a `computeSize(type)` method on the jBinary object, or perhaps a `computeSize` method on type descriptors which would produce this information when it's available.

**Comment by RReverser** [comment]
Yeah, I started smth similar earlier but thought that not much use cases actually need it (while the size of the library might blow).
